src/components/marker-chart/Canvas.js

```diff
@@ -59,7 +60,10 @@ const TEXT_OFFSET_TOP = 11;
 const TWO_PI = Math.PI * 2;
 const MARKER_DOT_RADIUS = 0.25;
 const TEXT_OFFSET_START = 3;
-const MARKER_LABEL_MAX_LENGTH = 150;
+
+// Export for tests.
+export const MARGIN_RIGHT = 15;
```

Comment:
TODO - This will be duplicated `MARGIN` definitions with #1270, I'd like to sneak in a shared place after landing the first one, so that both can share, or possibly follow-up with it.